### ISSUE TITLE
Labels on a hidden series cause performance problems

### ISSUE THREAD

**Issue by jyrimatti** [amcharts/amcharts5#894]
Hi,

using the most recent Amcharts (5.3.11), if I create a column series with ~1000 columns, each having an empty Bullet, and I try panning, the performance is fine:
```
series.bullets.push(() => {
        return am5.Bullet.new(root, {});
});
```

But if I add a Label to the bullets, the performance when panning becomes horrible (maybe around 5 fps on my M1):
```
series.bullets.push(() => {
        return am5.Bullet.new(root, {
            sprite: am5.Label.new(root, {
                text:         "{hello}",
                populateText: true
            })
        });
});
```

The curious thing here is, that I keep the series _hidden_ all the time (visible: false), so the chart shows no series data at all!

This is what Chrome tells me while panning for about 5 seconds:
<img width="512" alt="Screenshot 2023-04-21 at 21 28 23" src="https://user-images.githubusercontent.com/1271944/233709567-c69bf539-1177-41e6-b52c-0f68aa0fd6bb.png">

So what is happening, is that Amcharts is constantly calculating some bounds for labels on a hidden series.

This might be a nice and easy optimization if you can just find where to put the "if" condition.
It might be even nicer if Amcharts happens to calculate bounds all the time for all Labels (or is this even limited to Labels?) anywhere in the chart even if they are not visible (either in a hidden component or outside the viewport).

Thanks in advance, I'm just trying Amcharts5 after Amcharts4 and while the performance is maybe a bit worse, otherwise it's looking good!

**Comment by zeroin** [comment]
Thank you for the insights, we will take a look at it. By the way, you can always add isMeasured:false for the elements which size doesn't matter for you, this should improve performance a lot.

**Comment by martynasma** [comment]
Fixed in [5.3.12](https://github.com/amcharts/amcharts5/releases/tag/5.3.12).

## [5.3.12] - 2023-05-05

### Added
- New locale: Faroese (fo_FO). Thanks Martin Puge.

### Changed
- The `min` and `max` settings are now optional for `IHeatRule` ([Issue 883](https://github.com/amcharts/amcharts5/issues/883)).
- Removed `xlsx` package (with vulnerability) from dependencies in favor of a bundled hard copy (version 0.19.3) due to their decision not to publish to NPM. ([Issue 897](https://github.com/amcharts/amcharts5/issues/897)).

### Fixed
- Events now work properly with open shadow DOM (but not closed shadow DOM).
- Fixed export of data freezing in some cases where there are visual elements on the chart that are constantly being updated.
- Using parentheses in the in-line number formatting function was not working as needed.
- Tapping on a touch display outside chart was applying default state to elements that weren't being "touch-hovered".
- `getDataItemById()` method was not working properly on `Hierarchy` series.

[Full change log](https://github.com/amcharts/amcharts5/blob/master/packages/shared/CHANGELOG.md).

[Download options](https://www.amcharts.com/download/).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
